app/src/lib/stores/app-store.ts

```diff
@@ -999,7 +999,9 @@ export class AppStore extends TypedBaseStore<IAppState> {
           this.emitUpdate()
         })
 
-        const cleanup = () => (this.currentMergeTreePromise = null)
```

Comment:
I honestly thought this was something prettier did, so I accepted it.

Reply:
Yeah, I'm surprised it didn't have issues either. But to make doubly-sure it's considered a function I switched it.